static/bootstrap.js

```diff
@@ -113,10 +113,8 @@ const Wikicite = {
         .getInterface(Components.interfaces.nsIDOMWindow)
 
       domWindow.addEventListener('load', function listener() {
-        domWindow.removeEventListener('load', listener, false)
-        if (domWindow.document.documentElement.getAttribute('windowtype') == 'navigator:browser') {
```

Comment:
Just out of curiosity, why do you need to remove this conditional here? Is it because the preferences dialog does not meet the condition?

See my suggestion on the other thread, about passing the Prefs object as argument, instead of loading the whole Wikicite object on the preferences window. If that works, maybe you could restore this conditional?

I'm not sure about the consequences of loading this in windows other than those meeting the `navigator:browser`-type requirement.

Reply:
Yes, I needed to remove the conditional because the preferences dialog did not meet the condition. 

As you suggested, I've now changed it to pass the Prefs object directly through the window arguments and reverted the removal of the conditional. 

But I am wondering what the intended use for this code is, if it is not for making the Wikicite object available in other windows. Is it just for the main Zotero window?